frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_apply-template/readme.md

```diff
@@ -0,0 +1,300 @@
+                ]
+            }
+        ]
+    }
+}'
+```
+
+```RPC Response, Status: 200
+```
+
+### Failed Example
+
+Failed application of the template 'TEMP1' - template doesn't exist.
+
+```bash RPC Request
+curl --location --request PUT 'http://localhost:8181/rests/operations/template-manager:apply-template' \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "input": {
+        "template-node-id": "TEMP1",
+        "uniconfig-node": [
+            {
+                "uniconfig-node-id": "R1",
+                "variable": [
+                    {
+                        "variable-id": "remote-ip",
+                        "leaf-value": "172.30.15.1"
+                    },
+                    {
+                        "variable-id": "mode",
+                        "leaf-value": "vpn"
+                    }
+                ]
+            }
+        ]
+    }
+}'
+```
+
+```json RPC Response, Status: 404
+{
+  "errors": {
+    "error": [
+      {
+        "error-type": "application",
+        "error-tag": "data-missing",
+        "error-message": "Template with node ID 'TEMP1' is not present in CONFIG datastore.",
+        "error-info": {
+          "node-id": "R1"
+        }
+      }
+    ]
+  }
+}
+```
+
+### Failed Example
+
+Failed application of the template 'service\_group' to 2 UniConfig nodes
+- 'R1' and 'R2' - user hasn't provided values for all required
+  variables.
+
+```bash RPC Request
+curl --location --request PUT 'http://localhost:8181/rests/operations/template-manager:apply-template' \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "input": {
+        "template-node-id": "REDUNDANCY_TEMPLATE",
+        "uniconfig-node": [
+            {
+                "uniconfig-node-id": "R1",
+                "variable": [
+                    {
+                        "variable-id": "local-ip",
+                        "leaf-value": "172.30.15.1"
+                    },
+                    {
+                        "variable-id": "redundant-mode",
+                        "leaf-value": "service"
+                    }
+                ]
+            },
+            {
+                "uniconfig-node-id": "R2",
+                "variable": [
+                    {
+                        "variable-id": "local-ip",
+                        "leaf-value": "172.30.15.1"
+                    },
+                    {
+                        "variable-id": "redundant-mode",
+                        "leaf-value": "service"
+                    },
+                    {
+                        "variable-id": "min-interval",
+                        "leaf-value": 100
+                    }
+                ]
+            }
+        ]
+    }
+}'
+```
+
+```json RPC Response, Status: 500
+{
+  "errors": {
+    "error": [
+      {
+        "error-type": "application",
+        "error-tag": "operation-failed",
```

Comment:
it should not be system error (500) - in this case, user did not provide value for some variable

Reply:
Done.